tests/snmp/memory.py

```diff
@@ -1,7 +1,38 @@
 #!/usr/bin/python
 
 import time
-load = []
-for i in range(0, 1000):
-    load.append([' ' * 512000])
-    time.sleep(0.01)
+import subprocess
+
+
+def get_free_memory():
+    command = "grep MemFree /proc/meminfo | awk '{print $2}'"
+    output = subprocess.check_output(command, shell=True)
+    free_memory = int(output.strip())
+    return free_memory
+
+
+free_memory = get_free_memory()
+total_chars = 512000000
+
+if (free_memory * 1024) > int(total_chars * 130 / 100):
+    load = []
+    count = 1000
+    for i in range(0, count):
+        load.append([' ' * int(total_chars / count)])
```

Comment:
printing the load array is being done only in else block, what is the reason for that?

Reply:
The code in if block is the latest code in github, I try not to affect or modify the original code.
the code was modified by the PR https://github.com/sonic-net/sonic-mgmt/pull/9074.
If you think the printing here is necessary, I can add it back, thanks